packages/core/src/extensions/SlashMenu/SlashMenuExtension.ts

```diff
@@ -16,6 +18,7 @@ export const SlashMenuExtension = Extension.create<SlashMenuOptions>({
   addOptions() {
     return {
       commands: defaultCommands,
+      suggestionsMenuFactory: undefined, // TODO: fix undefined
```

Comment:
I think they should be always passed in, and set to a default when using blocknote/react.

but in that case, we should fix the typing so that suggestionsMenuFactory! (exclamation mark) a little further in the file is not necessary